yt/visualization/plot_modifications.py

```diff
@@ -3065,6 +3125,10 @@ def __call__(self, plot):
         lic_data = lic_data / lic_data.max()
         lic_data_clip = np.clip(lic_data, self.lim[0], self.lim[1])
 
+        if plot._swap_axes:
+            lic_data_clip = lic_data_clip.transpose()
```

Comment:
this line doesn't look right. I can't convince myself that transposition is equivalent to axes swapping (though it may _look_ equivalent with highly symmetric data)

Reply:
closing this thread as not actionnable. Results are correct; it doesn't matter that the code is slightly confusing to me